src/components/ec-filter-popover/ec-filter-popover.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div
+    class="ec-filter-popover"
+    data-test="ec-filter-popover"
+  >
+    <ec-popover
+      popover-class="ec-popover"
+      placement="bottom"
+      trigger="click"
+      level="modal"
+      @update:open="clickTrigger"
+    >
+      <div class="ec-filter-popover__filter-item">
```

Comment:
"Item" should be an item of a list. This div is a trigger which opens the popover content, I'd name it `ec-filter-popover__trigger`.